devicemodel/vmcfg/apl-mrb/vm1/Kconfig

```diff
@@ -1,3 +1,8 @@
+config DEFCONFIG_LIST
+	string
+	option defconfig_list
+	default "apl-mrb/vm1/configs/default.config"
```

Comment:
This folder should also be changed to `apl_mrb/vm1/...` to be consistent with the rest of this patch